R/resample.R

```diff
@@ -103,7 +106,8 @@ resample = function(learner, task, resampling, measures, weights = NULL, models
 
   rin = resampling
   more.args = list(learner = learner, task = task, rin = rin, weights = NULL,
-    measures = measures, model = models, extract = extract, show.info = show.info)
+    measures = measures, model = models, extract = extract, show.info = show.info,
+    cache = cache)
```

Comment:
Probably not necessary to pass the `cache` argument down as it's in `learner`.

Reply:
Same as above - if I remove it I face errors that cache is not populate through the function calls.